service/debugger/debugger.go

```diff
@@ -29,6 +29,8 @@ import (
 // lower lever packages such as proc.
 type Debugger struct {
 	config *Config
+	// arguments to launch a new process.
```

Comment:
What's the reason for moving this here from config?

Reply:
I just thought it'd be strange to modify a config. I can move it back to config if you wish.

Reply:
IMHO it's fine but let's hear what @derekparker thinks.

Reply:
I'm fine with it, but let's keep it named `processArgs`.